frontend/src/Components/ContentComponent.tsx

```diff
@@ -197,6 +208,56 @@ function updateMail(mail: HTMLSpanElement,
     });
 }
 
+function updateInternalExpandButton(expandButton: HTMLElement, appState: AppState) {
```

Comment:
I don't like this approach at all - it works but stinks horribly.

Reply:
Are there any concerns with it beside aesthetics? 

Reactifying all content (like here: https://github.com/spaceshelter/orbitar/pull/77) could be a performance problem, but I guess we could do this selectively.